scripts/vsmlrt.py

```diff
@@ -528,6 +541,22 @@ def RealESRGAN(
             "RealESRGANv2",
             "realesr-animevideov3.onnx"
         )
+    elif model in [5000, 5001, 5002, 5003, 5004]:
+        if model == 5000:
```

Comment:
```python3
f"{tuple(RealESRGANv2Model.__members__)[model]}.onnx".replace('_', '-')
```
is less verbose

Reply:
https://github.com/AmusementClub/vs-mlrt/pull/42#issuecomment-1515751375